cg/apps/demultiplex/sample_sheet/create.py

```diff
@@ -22,22 +20,14 @@ def get_sample_sheet_creator(
     lims_samples: List[FlowCellSample],
     force: bool,
 ) -> SampleSheetCreator:
-    """Returns an initialised sample sheet creator according to the flow cell sequencer."""
-    sequencer: str = flow_cell.sequencer_type
-    if sequencer == Sequencers.NOVASEQ:
+    """Returns an initialised sample sheet creator according to the software used for demultiplexing."""
+    if bcl_converter == BclConverter.BCL2FASTQ:
         return SampleSheetCreatorV1(
             bcl_converter=bcl_converter, flow_cell=flow_cell, lims_samples=lims_samples, force=force
         )
```

Comment:
I am not sure if this is correct. I think if we do this change we would also need to change the SampleSheetCreatorV1. Currently, this is a bug in production - we cannot handle BCL2FASTQ sample sheets as they were once generated using the `demultiplexing` repo. We could probably modify SampleSheetCreatorV1 to represent the BCL2FASTQ sample sheets. However, we should probably with that change call it SampleSheetCreatorBcl2Fastq and SampleSheetCreatorBclConvert

Reply:
fixed